APE1.rst

```diff
@@ -1,11 +1,11 @@
 APE Purpose and Process
 -----------------------
 
-author: Perry Greenfield
+author: Perry Greenfield, Lia Corrales, Thomas Robitaille
```

Comment:
wasn't the conclusion that the addendums get a new metadata header with the new authors rather than rewriting history for the original one?

Reply:
Yes indeed, I've implemented that now